src/EnergyPlus/MixedAir.cc

```diff
@@ -3758,8 +3758,8 @@ namespace MixedAir {
 
         } // if (AirLoopNum > 0)
 
-        // Set the relief air flow rate (must be done last to account for changes in OAMassFlow
-        this->RelMassFlow = max(this->OAMassFlow - this->ExhMassFlow, 0.0);
+          // Set the relief air flow rate (must be done last to account for changes in OAMassFlow
+        this->RelMassFlow = this->OAMassFlow - this->ExhMassFlow;
```

Comment:
Big question here is if relief air can be negative. Not sure where this limitation came from. Seems reasonable to expect no intake into relief node but this issue raises questions.